libraries/message/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "dora-message"
 # versioned separately from the other dora crates
-version = "0.4.3"
+version = "0.4.4"
```

Comment:
I think we should have bumped this to `v0.5.0` because there were breaking changes to the messages since `v0.4.3`. Our version compatibility logic that we added in https://github.com/dora-rs/dora/pull/614 only works if we follow semver for the `dora-message` crate. If we don't, we will get random deserialization errors instead of the "mismatched message format" error.

It's too late now to fix this, but we should keep it in mind for the next dora release.